src/styles/_catalog.scss

```diff
@@ -1,12 +1,13 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
   max-width: 944px;
   margin: 0 auto;
+  justify-content: center;
```

Comment:
Remove 'justify-content: center;', as this property is not needed when using grid layout.